bootstrap/namespaces/helmfile/values/namespaces-wc.yaml.gotmpl

```diff
@@ -20,5 +20,8 @@ namespaces:
 {{ if .Values.user.alertmanager.enabled }}
 - name: alertmanager
 {{ end }}
+{{ if .Values.thanos.enabled }}
+- name: thanos
```

Comment:
Do we need:

```
  labels:
    admission.gatekeeper.sh/ignore: "true"
```

Otherwise, we need to make sure that trusted repos, requests and networkpolicies are fine in this namespace.

Reply:
We don't run gatekeeper in SC

Reply:
Right! Thanks for the refresher.